### ISSUE TITLE
Implement a social link view component

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#34]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

To render social links regardless of view models.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #19 

### Blockers

This issue is blocked by the following issue(s): #`issue number`
